### PR TITLE
Billy/verification

### PR DESCRIPTION
**Pull request by billyfurie**
**Verification API**

This endpoint handles the verification codes for account verification, resetting password, and any general authentication using a code sent to the user.

Allows for creation of the code (which entails sending the code to the user via email or phone
Allows for verification of the code sent to the user

I have documentation currently in-progress in Postman for this and I will be happy to help with integrating of this endpoint into the frontend.

### CODE REVIEW

File: backend/routes/userRoutes.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 const express = require('express');
 const router = express.Router();
+
 const {
     registerUser,
-    loginUser,
+    loginUser
```
Comment (billyfurie) [inline backend/routes/userRoutes.js:6]: I actually don't know when this change was made. This could be left as the original.

**Comment by ryanata** [comment]
Pull main and resolve conflicts before merge.

**Review by ryanata — CHANGES_REQUESTED** [review]
Overall, looks good! Don't forget to resolve comments

**Review by ryanata — APPROVED** [review]
lgtm

**Pull request merged**